src/styles/blocks/header.scss

```diff
@@ -0,0 +1,19 @@
+.header {
+  &__container {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
```

Comment:
I don't see header's box-shadow, you need to fix it
![img1](https://user-images.githubusercontent.com/31217573/186367174-7be20ce3-c89d-4042-980c-0f362cb4cfa7.jpg)